AsyncAwait.Task1.CancellationTokens/Program.cs

```diff
@@ -46,16 +62,34 @@ private static void Main(string[] args)
         Console.ReadLine();
     }
 
-    private static void CalculateSum(int n)
+    private static async void CalculateSum(int n)
```

Comment:
do not use async void anywhere except for event handlers is windows apps, use Task